addons/gamification/data/goal_base.xml

```diff
@@ -20,76 +20,199 @@
         <record id="simple_report_template" model="mail.template">
             <field name="name">Simple Challenge Report Progress</field>
             <field name="model_id" ref="gamification.model_gamification_challenge"/>
-            <field name="body_html"><![CDATA[
-<p>
-    <strong>${object.name}</strong>
-</p>
-<p class="oe_grey">The following message contains the current progress for the challenge ${object.name}</p>
-
-% if object.visibility_mode == 'personal':
-    <table width="100%" border="1">
-    <tr>
-        <th>Goal</th>
-        <th>Target</th>
-        <th>Current</th>
-        <th>Completeness</th>
-    </tr>
-    % for line in ctx["challenge_lines"]:
-        <tr
-            % if line['completeness'] >= 100:
-                style="font-weight:bold;"
-            % endif
-            >
-            <td>${line['name']}</td>
-            <td>${line['target']}
-            % if line['suffix']:
-                ${line['suffix']}
-            % endif
-            </td>
-            <td>${line['current']}
-            % if line['suffix']:
-                ${line['suffix']}
-            % endif
-            </td>
-            <td>${line['completeness']} %</td>
-        </tr>
-    % endfor
-    </table>
-% else:
-    % for line in ctx["challenge_lines"]:
-        <table width="100%" border="1">
-            <tr>
-                <th colspan="4">${line['name']}</th>
-            </tr>
-            <tr>
-                <th>#</th>
-                <th>Person</th>
-                <th>Completeness</th>
-                <th>Current</th>
-            </tr>
-            % for goal in line['goals']:
-                <tr
-                    % if goal.completeness >= 100:
-                        style="font-weight:bold;"
-                    % endif
-                    >
-                    <td>${goal['rank']}</td>
-                    <td>${goal['name']}</td>
-                    <td>${goal['completeness']}%</td>
-                    <td>${goal['current']}/${line['target']}
-                    % if line['suffix']:
-                        ${line['suffix']}
+            <field name="subject">New badge granted</field>
```

Comment:
Why is the "Simple Challenge Report Progress" subject called "New badge granted"?